src/App.js

```diff
@@ -1,79 +1,15 @@
 import React from 'react';
 import './App.css';
+import Welcome from './components/Welcome-page/Welcome-page';
```

Comment:
компоненты должны быть названы камелкейсом, если более одного слова
```suggestion
import Welcome from './components/WelcomePage/WelcomePage';
```